### ISSUE TITLE
import images appears to be importing many 3rd party images

### ISSUE THREAD

**Issue by mrog** [aws/eks-anywhere#4256]
**What happened**:
Our customer is having an issue when using eks-a to import images into their registry.  The import process appears to be pushing many 3rd party images to the registry.  This makes it so anyone who imports the images needs to have permission to many registry namespaces, or else the process fails with a permission error.  But the list of required namespaces unknown, and could be large.

Sample error message:
```
Error: writing images to destination with image mover: image processor worker failed, rest of jobs were aborted: unauthorized: The client does not have permission for manifest: org.artifactory.exception.CancelException: mark_rogers3 does not have permissions to create modify or delete objects in namespace dockerv2-local:retailappsatx:retailappsatx
```

**What you expected to happen**:
Eks-a should only push images that are produced by the eks-a team.

**How to reproduce it (as minimally and precisely as possible)**:

```
eksctl anywhere download images --output "$TARGET_ARTIFACTS_FILE" -v 10 2>&1 | tee "$TMPDIR/download_images_output.txt"

grep 'Reading bundles manifest' "$TMPDIR/download_images_output.txt" | awk '{print $7}' | sed 's/}//g' | xargs -I {} curl -L -s {} > "$TARGET_BUNDLE_MANIFEST_FILE"

REGISTRY_USERNAME="$REGISTRY_USERNAME" REGISTRY_PASSWORD="$REGISTRY_PASSWORD" eksctl anywhere import images --input "$TARGET_ARTIFACTS_FILE" --registry $REGISTRY_URL --bundles "$TARGET_BUNDLE_MANIFEST_FILE" -v 10
```

**Anything else we need to know?**:

**Environment**:
- EKS Anywhere Release: v0.12.2
- EKS Distro Release: 0.115.0

**Comment by mrog** [comment]
As an experiment, I ran the same script using a local docker registry.Then I looked at the

**Comment by mrog** [comment]
As an experiment, I ran the same download and import on a Linux machine in my own environment.  I used `docker run -d -p 5000:5000 --name registry registry:2` to create a local registry.

At the end of the import, I didn't see any unfamiliar repositories at http://localhost:5000/v2/_catalog.  [Here is the list of repositories in the registry](https://github.com/aws/eks-anywhere/files/10137314/catalog.json.gz).
